app/logic/util/backend-wrapper.ts

```diff
@@ -0,0 +1,13 @@
+// #if [MOBILE]
```

Comment:
I'm sorry,   Jeremy! My comment must have gotten lost in the github UI. I wrote:

Could you please rename this file backend-wrapper to `os-integration.ts` ?

Sorry again...